### DISCUSSION TITLE
Possible issue(s) (or desired clarifications) with updated sktime Classification notebook

### DISCUSSION THREAD

**Discussion by jmwhyte** [alan-turing-institute/sktime#1956] [Q&A]
I refer to this page
https://github.com/alan-turing-institute/sktime/blob/main/examples/classification.ipynb
from which I have extracted commands.

In the cell beginning
`# Random train set for a two class problem with 10 instances of length 50`

arrow_train_X, arrow_train_y = load_arrow_head(split="train", return_type="numpy2d")
yields 
TypeError: load_arrow_head() got an unexpected keyword argument 'return_type'

Also 
`hc2.fit(X_train, y_train)`
leads to 
` TypingError: Failed in nopython mode pipeline (step: nopython frontend)
non-precise type array(pyobject, 1d, C)
During: typing of argument at /Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/sktime/transformations/panel/shapelet_transform.py (422)

File "../../../../../../../../../../Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/sktime/transformations/panel/shapelet_transform.py", line 422:
    def _find_shapelet_quality(
        <source elided>
    ):
        orderline = [] `

Finally under "Multivariate classification""

"Many classifiers, including ROCKET and HC2, are configured to work with multivariate input."
Is it possible to specify these, so that we don't spend time using a classifer that is not designed for multivariate use?

**Comment by fkiraly** [comment]
@TonyBagnall, FYI - did you not refactor this code recently?

**Reply by fkiraly** [reply]
@TonyBagnall?
